cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/RollClient.java

```diff
@@ -50,6 +50,14 @@ interface RollClient {
      */
     Map<String, Integer> describeTopicMinIsrs(List<String> topicNames);
 
+    /**
+     * Describe the metadata quorum info.
+     * @return A map from controller quorum followers to their {@code lastCaughtUpTimestamps}.
+     * @throws io.strimzi.operator.common.UncheckedExecutionException
+     * @throws io.strimzi.operator.common.UncheckedInterruptedException
+     */
+    Map<Integer, Long> describeQuorumState();
```

Comment:
If this is returning the `lastCaughtUpTimestamps` the method would be better called something like `quorumLastCaughtUpTimestamps`.